features/unit/v2algodclient_paths.feature

```diff
@@ -179,3 +179,19 @@ Feature: Algod REST API v2 Paths
     Examples:
       | path            | method |
       | /v2/ledger/sync | delete |
+
+  @unit.timestamp
+  Scenario Outline: SetBlockTimeStampOffset path
+    When we make a SetBlockTimeStampOffset call against offset <offset>
+    Then expect the path used to be "<path>"
```

Comment:
May as well use the new one :shrug:
```suggestion
    Then expect the request to be "get" "<path>"
```

Reply:
Updated in : [9ec4a57](https://github.com/algorand/algorand-sdk-testing/pull/276/commits/9ec4a577a4df9c65d140dd8cc8c6ccbdd9f425c4)